packages/sanity/src/core/studio/components/navbar/new-document/NewDocumentButton.tsx

```diff
@@ -170,7 +170,7 @@ export function NewDocumentButton(props: NewDocumentButtonProps) {
     return (
       <InsufficientPermissionsMessage
         currentUser={currentUser}
-        operationLabel="create any document"
+        action={t('insufficient-permissions-message.action.create-any-document')}
```

Comment:
I think a better approach here is to use the `operationLabel` to pass a context to the translation key